guestbook/php-redis/Makefile

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TAG = v5
+REGISTRY = gcr.io/google-samples
+ARCH ?= $(shell go env GOARCH)
+ALL_ARCH = amd64 arm arm64 ppc64le
+
+QEMUVERSION=v2.7.0
+
+IMAGE = $(REGISTRY)/gb-frontend
+MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
+
+# Set default base image dynamically for each arch
+ifeq ($(ARCH),amd64)
+	BASEIMAGE=php:5-apache
+endif
+ifeq ($(ARCH),arm)
+	QEMUARCH=arm
+	BASEIMAGE=armhf/php:5.6-apache
```

Comment:
These are official images, here is the information - https://github.com/docker-library/official-images#architectures-other-than-amd64